routesrv/options.go

```diff
@@ -100,6 +100,10 @@ type Options struct {
 	// used with external name services (type=ExternalName).
 	KubernetesAllowedExternalNames []*regexp.Regexp
 
+	// KubernetesForceService overrides the default Skipper functionality to route traffic using Kubernetes Endpoints,
+	// instead using Kubernetes Services.
+	KubernetesForceService bool
```

Comment:
I chose to deviate from the `ForceKubernetesService` naming here, as this is not the `Options` struct relating to Kubernetes configuration, and the Kubernetes prefix seems more consistent with the other fields.

Reply:
Yes very good!
Can you add some docs into docs/kubernetes/ingress-controller.md to let people understand use case and how to configure?
I think code and tests are good to go.

Reply:
Maybe we could use `KubernetesUseServiceDomainName` to avoid negative connotation of "Force"?

Reply:
I prefer Force, because it's more clear that you override the default choice.